documentation/content/codingstyle.rst

```diff
@@ -180,6 +180,12 @@ $PATH, you should use 'type ', instead of 'which '.
 The output of 'which' is not machine parsable and its exit code
 is not reliable across platforms.
 
+Sourcing files
```

Comment:
Maybe something like:
Do not source code using `.` or `source`. We have a helper function for that named `include`...

Also, we don't need a single commit for this change. Please, squash it with the first commit.

Reply:
Also, avoid adding extra white space at the end of the line.